### PR TITLE
Update navigation logo

### PR DESCRIPTION
**Pull request by tammersaleh**
## Changes proposed in this pull request:

- Fix navigation logo
- Clean up documentation

### Todo:

- [ ] ~~Change `pipeline.yml` to deploy using the `cf` cli instead of `cf-resource`, which uses autopilot (since it's leaving `-venerable` apps hanging around)~~
- [x] Determine what to do about currently failing `master` build.  Believe this is due to some issue in upstream stratos.
- [x] Run the pipeline through staging and observe the glorious new logo
- [x] Continue to prod

## security considerations

None

### CODE REVIEW

File: ci/pipeline.yml [hunk 1]
```diff
@@ -131,15 +147,25 @@ resources:
...
+    branch: master
+    # tag_filter: stable
+    # branch: v2-master
 
 - name: stratos-config
   type: git
   source:
     uri: https://github.com/18F/cg-deploy-stratos.git
-    branch: master
+    branch: tsaleh/1304-cg-logo
```
Comment (bengerman13) [inline ci/pipeline.yml]: this should be master now, right?

File: manifest.yml [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
   - name: stratos
-    host: dashboard-beta
+    host: dashboard
     memory: 1512M
     disk_quota: 1024M
     timeout: 180
-    buildpack: https://github.com/cloudfoundry-incubator/stratos-buildpack#v2.4
+    # Buildpack pin should match upstream manifest:
+    # https://github.com/cloudfoundry/stratos/blob/stable/manifest.yml
+    buildpack: https://github.com/cloudfoundry/stratos-buildpack#v2.4
```
Comment (bengerman13) [inline manifest.yml]: is there a way we could extract this?

Comment (tammersaleh) [reply manifest.yml]: extract the comment, or extract the version of the buildpack?  can you go into more detail?

File: ci/pipeline.yml (hunk 1 above)
Comment (tammersaleh) [reply ci/pipeline.yml]: very good catch!

File: manifest.yml (hunk 2 above)
Comment (bengerman13) [reply manifest.yml]: the buildpack version

**Pull request merged**